sklearn/hmm.py

```diff
@@ -1266,7 +1266,7 @@ def _accumulate_sufficient_statistics(self, stats, obs, framelogprob,
                     cvnorm = np.copy(norm)
                     shape = np.ones(tmp_gmm.covars_.ndim)
```

Comment:
Just curious, what's the policy on hmm.py given that it has been deprecated and moved outside of scikit-learn? I naively thought that there was no point touching it any more.